src/c++/perf_analyzer/infer_context.cc

```diff
@@ -258,7 +260,11 @@ InferContext::AsyncCallbackFuncImpl(cb::InferResult* result)
         if (thread_stat_->cb_status_.IsOk() == false) {
           return;
         }
-        it->second.response_times_.push_back(std::chrono::system_clock::now());
+        it->second.response_timestamps_.push_back(
+            std::chrono::system_clock::now());
+        std::string response_output{""};
+        result->Output(response_output);
+        it->second.response_outputs_.push_back(response_output);
```

Comment:
Currently, this puts an empty string in the list of response outputs for a request if that request receives a null final response. We _do_ record a timestamp for a null final response. Should we also record an empty string output for a null final response?

Reply:
When you write out the profile_export file, we need something to write no?
I think an empty string is safe here.

Reply:
Since we dump based on outputs in the response now, the null responses get excluded by default.

Reply:
I didn't quite understand.

Reply:
I think that is correct

Reply:
I think it would be useful for debugging / understanding to have a corresponding `{}` for a timestamp representing a null response - i.e. have a set of outputs corresponding to every timestamp. Any tool ingesting this information can decide to make use of the `{}` empty output or not. They can also use this information to decide if they disregard the corresponding timestamp or not.

Without this information, I think the ingester of this information has no idea if a timestamp corresponds to a "real" response or a "null final" one.